libcore/gof-file.c

```diff
@@ -1354,7 +1384,11 @@ gof_file_get (GFile *location)
     GOFFile *file = NULL;
     GOFDirectoryAsync *dir = NULL;
 
-    g_return_val_if_fail (location != NULL && G_IS_FILE (location), NULL);
+    //g_return_val_if_fail (location != NULL && G_IS_FILE (location), NULL);
+
+    if (location == NULL) {
+
```

Comment:
something is missing here I guess

Reply:
Not sure what I was going to do here ... I think return a GOF.File.get_null () rather than NULL.